OsmAnd/res/values/strings.xml

```diff
@@ -106,12 +106,12 @@
     <string name="map_center_widget">Coordinates: Map center</string>
     <string name="current_location_widget">Coordinates: Current location</string>
     <string name="external_sensors_plugin_name">External Sensors</string>
-    <string name="external_sensors_plugin_description">Monitoring of heart rate, bicycle speed, bicycle power, and cadence. Requires your device to be connected to respective sensors via the ANT+ wireless personal network protocol.</string>
+    <string name="external_sensors_plugin_description">Monitoring of heart rate, bicycle speed, power, and cadence via ANT+ connection to respective sensors.</string>
     <string name="map_widget_altitude_map_center">Elevation: map center</string>
-    <string name="map_widget_altitude_map_center_desc">Shows the elevation above sea level of the current map center.</string>
+    <string name="map_widget_altitude_map_center_desc">Shows elevation above sea level of the current map center.</string>
     <string name="map_widget_altitude_current_location">Altitude: current location</string>
-    <string name="map_widget_altitude_current_location_desc">Shows the altitude above sea level of the current geolocation.</string>
-    <string name="map_widget_altitude_desc">Display the altitude of the current geolocation or the elevation of the current map center</string>
+    <string name="map_widget_altitude_current_location_desc">Shows elevation above sea level of the current geolocation.</string>
```

Comment:
Articles are needed.

Line 113: Unless I am mistaken, this is refers to the measured value (not necessarily on the ground), hence it is altitude.